code/modules/clothing/under/color.dm

```diff
@@ -233,3 +233,10 @@
 	icon_state = "rainbow_skirt"
 	item_state = "rainbow"
 	can_adjust = FALSE
+
+/obj/item/clothing/under/color/rilena
```

Comment:
Cut the shirt out of the pants, and add it to the undershirt selection.

Reply:
Good idea. Will do.

Reply:
You can also use the existing shirts as a shading basis, it'd help.